tutorials/nvidia-triton/index.mdx

```diff
@@ -0,0 +1,117 @@
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key/)
+  - You have [created GPU Instance](/compute/instances/how-to/create-an-instance/) that runs Ubuntu GPU 12
+  - You have [activated your observalibility Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+  - You have sudo privileges or access to the root user
+  - You have basic understanding of machine learning models and Docker.
+
+</Message>
+
+
+## Storing models in Scaleway Object Storage
+
+In this step, we will create an Object Storage bucket on Scaleway and upload your machine learning model. If you have already created and uploaded your model to an Object Storage bucket, you can skip this step.
+For this tutorial, we will use a pre-trained model available in the Triton Inference Server model repository. You can replace this with your own model if desired.
+
+1. Click **Object Storage** on the left side menu of the console. The Object Storage dashboard displays.
+2. [Create a new bucket](/storage/object/how-to/create-a-bucket/) to store your model artifacts.
+3. Download the example model repository from NVIDIA's GitHub to your local computer:
+   ```bash
+   git clone https://github.com/triton-inference-server/server.git
+   ```
+4. Download the example models:
+    ```bash
+    cd server/docs/examples
+    ./fetch_models.sh
+    ```
+5. Navigate to the `server/docs/examples/model_repository` directory within the cloned repository.
+6. Upload the example model folder to your bucket in Scaleway Object Storage. You can use the [Scaleway Object Storage API](/docs/storage/object/api-cli/using-api-call-list/), any S3 compatible tool, or web interface to upload the model folder.
+    <Message type="tip">
+      You can use the `s3cmd` [command-line tool](/docs/tutorials/s3cmd/) or any other S3 compatible tool to upload your data.
+    </Message>
+
+## Configuring Triton Inference Server
+
+1. Log into your GPU Instance using SSH.
+    ```
+    ssh root@<gpu_instance_ip>
+    ```
+3. Edit the Triton server configuration file, typically named `config.pbtxt`. This file should be located in the Triton Inference Server installation directory.
+4. Configure the model repository path by adding or modifying the `model-repository` parameter:
+   ```bash
+   model-repository: "s3://https://s3.fr-par.scw.cloud:443/<your-bucket-name>/<path-to-model-folder>"
+   ```
+   Replace `<your-bucket-name>` and `<path-to-model-folder>` with the appropriate values for your Scaleway Object Storage setup.
+   <Message type="tip">
+     When using S3, the credentials and default region can be passed by using either the [created a AWS-CLI configuration file](/storage/object/api-cli/object-storage-aws-cli/) or via the respective environment variables. If the environment variables are set they will take a higher priority and will be used by Triton instead of the credentials set using the AWS-CLI configuration file.
```

Comment:
```suggestion
     When using S3, the credentials and default region can be passed by using either the [created AWS-CLI configuration file](/storage/object/api-cli/object-storage-aws-cli/) or via the respective environment variables. If the environment variables are set they will take a higher priority and will be used by Triton instead of the credentials set using the AWS-CLI configuration file.
```